docs/overview.md

```diff
@@ -0,0 +1,72 @@
+
+Infrastructure
+--------------
+
+![SecureDrop architecture overview diagram](./SecureDrop_diagram.png)
+
+There are four main components of SecureDrop: the servers, the administrators, the sources, and the journalists.
+
+
+### Servers
+
+At SecureDrop's heart is a pair of severs: the *Application (“App”) Server* which runs the core SecureDrop software, and the *Monitor (“Mon”) Server* which keeps track of the *Application Server* and sends out alerts if there's a problem. These two servers run on dedicated hardware connected to a dedicated firewall appliance. They are typically located physically inside the newsroom.
+
+
+### Systems Administrator
+
+The SecureDrop servers are managed by a systems administrator; for larger newsrooms, there may be a team of systems administrators. The administrator uses a dedicated *Admin Workstation* running [Tails][] and connects to the App and Mon servers over authenticated [Tor Hidden Services][] and manages them using [Ansible][].
+
+
+### Source
+
+A source submits documents and messages by using [Tor Browser][] (or Tails) to access the *Source Interface*: a public Tor Hidden Service. Submissions are encrypted in place on the App server as they are uploaded.
+
+
+### Journalists
+
+Journalists working in the newsroom use two machines to interact with SecureDrop. First, they use a *Journalist Workstation* running Tails to connect to the *Document Interface*, an authenticated Tor Hidden Service. Journalists download [GPG][]-encrypted submissions and copy them to a *Transfer Device* (a thumb drive or DVD). Those submissions are then connected to the airgapped *Secure Viewing Station* (SVS) which holds the key to decrypt them. Journalists can then use the SVS to read, print, and otherwise prepare documents for publication. Apart from those deliberately published, decrypted documents are never accessed on an Internet-connected computer.
+
+* * * * *
+
+The terms in italics are terms of art specific to SecureDrop. The [Terminology Guide](./terminology.md) provides more-precise definitions of these and other terms. SecureDrop is designed against a comprehensive [threat model](./thread-model.md), and has a specific notion of the [roles](./roles.md) that are involved in its operation.
+
+
+Operation
+---------
+
+### Planning & Preparation
+
+Setting up SecureDrop is a multi-step process. Before getting started, you should make sure that you're prepared to operate and maintain it. You'll need a systems administrator who's familiar with Linux, the GNU utilities, and the Bash shell. You'll need the [hardware](./hardware.md) on which SecureDrop runs — this will normally cost $2000-$3000 dollars. The journalists in your organization will need to be trained in the operation of SecureDrop, and you'll need to publish and promote your new SecureDrop instance afterwards — using your existing websites, mailing lists, and social media.
```

Comment:
If you literally buy everything new, this range is correct; in fact, it might be a little low. The last install that I helped price out ended up being $3300 (and they recycled a machine for the SVS instead of buying one), but that also included Journalist Workstation materials for 10 journalists, which is more than average.

We should probably just create an example "buy order" spreadsheet on Google Docs or something (we can base it off of CPJ's, since they shared it with us) and link to it here. Much more useful than a table in a markdown/ReST document.